style.css

```diff
@@ -0,0 +1,581 @@
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: lato, sans-serif;
+}
+
+section {
+  padding: 1rem;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  text-transform: capitalize;
+}
+
+.upper {
+  display: none;
+}
+
+header {
+  background-image: url(images/bgtest.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 1rem;
+  padding-bottom: 4rem;
+}
+
+.logo {
+  display: none;
+}
+
+.icon i {
+  font-size: 30px;
+  color: #272a31;
+}
+
+nav ul {
+  position: absolute;
+  height: 100vh;
+  width: 60%;
+  background-color: rgb(235, 230, 230);
+  top: 0;
+  right: -100%;
+  padding: 2rem 1rem;
+  display: none;
+  transition: 0.5s linear;
+  z-index: 99;
+}
```

Comment:
- [ ] For the mobile menu, kindly make the `width` `100%` or `100vw` and the `background-color` transparent or the same colour as the top section of the page. This way, your mobile menu will look very professional. Kindly implement this on both the _**Home**_ and _**About**_ Pages 👍